core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4411,7 +4411,7 @@ public String toString() {
           .build();
   public static final PropertyKey USER_METRICS_COLLECTION_ENABLED =
       new Builder(Name.USER_METRICS_COLLECTION_ENABLED)
-          .setDefaultValue(false)
+          .setDefaultValue(true)
```

Comment:
This change will make the default behavior changed, the user report metric will harm for performance, do you really need to do this change while users do not aware this change?

Reply:
@apc999 @yuzhu

Reply:
so far we have not seen much perf degradation (maybe 5-10%)? do you have specific instances where this makes perf much worse? @maobaolong

Reply:
one reason we want to make this default true, is users get confused why there is no local read metric when they just use it out of the box. we want to make the default experience better.